topostats/validation.py

```diff
@@ -56,6 +56,8 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
             ".ibw",
             ".gwy",
             ".topostats",
+            ".stp",
+            ".top",
             error="Invalid value in config for 'file_ext', valid values are '.spm', '.jpk', '.ibw', '.gwy', '.topostats', or '.asd'.",
```

Comment:
We should add these options into the error message too...

```suggestion
            error="Invalid value in config for 'file_ext', valid values are '.spm', '.jpk', '.ibw', '.gwy', '.topostats', '.asd', "
            '.stp' or '.top'.",
```